finance.reckoner.Reckoner.yaml

```diff
@@ -0,0 +1,33 @@
+app-id: finance.reckoner.Reckoner
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+command: reckoner
+separate-locales: false
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=home
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: reckoner
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/bin ${FLATPAK_DEST}/reckoner ${FLATPAK_DEST}/share/icons/hicolor/192x192/apps/ ${FLATPAK_DEST}/share/applications/ ${FLATPAK_DEST}/share/appdata/
+      - cp reckoner-git/linux/flatpak/${FLATPAK_ID}.png ${FLATPAK_DEST}/share/icons/hicolor/192x192/apps/
```

Comment:
192x192 isn't a valid icon size.

Reply:
Would this be the cause of the appstream failure?

Reply:
```
Error loading desktop file: Failed to find icon: Failed to find icon finance.reckoner.Reckoner
```

correlation is not causation, but I can tell you appstreamcli isn't looking for a 192x192 icon.

Reply:
The short answer is yes, it was causing the issue.

Interestingly 192x192 works in MOST cases. It worked in general for showing up in the menu with the icon and desktop handling it correctly. It does not work in all cases and appstream was where it failed.